examples/drivenet/drivenet_config.yaml

```diff
@@ -0,0 +1,66 @@
+# Config format schema number
```

Comment:
Would it make sense to have a common config folder?
Now we have at least 3 (same) configs across the examples - or is that badly overloaded due to model_params being once for SimNet and once for DriveNet?

Reply:
they are slightly different so I think we should leave them in different folder for now. Definitely something to consider for later though